packages/vite-plugin-svelte/src/utils/resolve.ts

```diff
@@ -14,7 +14,13 @@ export function resolveViaPackageJsonSvelte(importee: string, importer?: string)
 }
 
 function isBareImport(importee: string): boolean {
-	if (!importee || importee[0] === '.' || importee[0] === '\0' || path.isAbsolute(importee)) {
+	if (
+		!importee ||
+		importee[0] === '.' ||
+		importee[0] === '\0' ||
+		importee.includes(':') ||
```

Comment:
Should this check if it starts with `node:`? I'm guessing we're unlikely to run into a `:` elsewhere, but I didn't know if there are other prefixes that you might see besides `node`?
```suggestion
		importee.startsWith('node:') ||
```

Reply:
I replied at https://github.com/sveltejs/vite-plugin-svelte/pull/225#discussion_r768003988

Reply:
ok. sounds good